app/src/main/java/com/babylon/wallet/android/domain/usecases/BuildDappResponseUseCase.kt

```diff
@@ -231,7 +230,8 @@ class BuildAuthorizedDappResponseUseCase @Inject constructor(
                                 oneTimeAccounts = oneTimeAccountsResponseItem.getOrNull(),
                                 ongoingAccounts = ongoingAccountsResponseItem.getOrNull(),
                                 ongoingPersonaData = ongoingSharedPersonaData?.toWalletToDappInteractionPersonaDataRequestResponseItem(),
-                                oneTimePersonaData = onetimeSharedPersonaData?.toWalletToDappInteractionPersonaDataRequestResponseItem()
+                                oneTimePersonaData = onetimeSharedPersonaData?.toWalletToDappInteractionPersonaDataRequestResponseItem(),
+                                proofOfOwnership = null
```

Comment:
Are we sure this should be null?

Reply:
This is here only until #1216  is merged